components/03-modules/dashboard-toolbar/_dashboard-toolbar.scss

```diff
@@ -0,0 +1,40 @@
+.dashboard-toolbar {
+    &__amount {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin-bottom: 15px;
+
+        @include mq($screen-m) {
+            justify-content: flex-start;
+        }
+    }
+
+    &__limiter {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin-bottom: 15px;
+
+        @include mq($screen-m) {
+            justify-content: flex-end;
+        }
+    }
+
+    &__pager {
+        margin-bottom: 0;
+    }
+
+    &__information {
+        margin: 0 15px;
+    }
+
+    .select {
+        width: 100px;
+        margin-bottom: 0;
+
+        .label {
+            margin-bottom: 0;
+        }
+    }
+}
```

Comment:
Did you try displays the toolbar without one of these elements? In Magento you can hide each of these and use only one or two.

Reply:
Yes, I tried and it displays in full width with content on right or left side

Reply:
👍